storage/innobase/xtrabackup/doc/source/advanced/page_tracking.rst

```diff
@@ -0,0 +1,97 @@
+
+2. Check whether the *mysqlbackup* component is installed succesfuly:
+
+  .. code-block:: bash
+
+     SELECT COUNT(1) FROM mysql.component WHERE component_urn='file://component_mysqlbackup';
+
+Using the page tracking
+-----------------------
+
+.. note::
+
+   To use the following options for page tracking, run the ``BACKUP_ADMIN`` privilege.
+
+1. After the *component_mysqlbackup* is loaded and active on the server, start page tracking:
+
+  .. code-block:: bash
+
+     SELECT mysqlbackup_page_track_set(true);
+
+2. Enable the page tracking functionality for the full and incremental backups with the following option:  
+
+``--page-tracking``
+
+It serves the dual purpose:
+
+* Reset page tracking at the start of the backup so that the next incremental backup can use page tracking.
+
+* Use page tracking for incremental backup if page tracking data is available from the backup start checkpoint LSN.
+
+3. To check the LSN value starting from which changed pages are tracked, use the following option:
+
+  .. code-block:: bash
+
+     SELECT mysqlbackup_page_track_get_start_lsn();
+
+Once the page tracking is enabled, the next incremental backup will use page tracking to find changed pages.
+
+First backup using page tracking
+--------------------------------
+
+For the very first full backup using page tracking, Percona XtraBackup may have a delay. The following is an example of the message you can receive: 
+
+  .. code-block:: bash
+
+     xtrabackup: pagetracking: Sleeping for 1 second, waiting for checkpoint lsn 17852922 to reach to page tracking start lsn 21353759
+
+This delay is required to ensure that the page tracking start LSN is more than the backup end checkpoint LSN which is required for the next incremental backup. If the server page flushing rate is low or it uses a very big buffer pool, you can start page tracking before creating the first backup. Thus, you ensure that page tracking LSN is more than the checkpoint LSN of the server. It is required for the first backup only.
+
+Uninstall the *mysqlbackup* component
+-------------------------------------
+
+If you need to uninstall the *mysqlbackup* component, use the following option:
+
+  .. code-block:: bash
+
+     UNINSTALL COMPONENT "file://component_mysqlbackup"
+
+Stop page tracking
+------------------
+
+If you need to stop page tracking, use the following option:
+
+  .. code-block:: bash
+
+     SELECT mysqlbackup_page_track_set(false);
+
+Known issues
+------------
+
+1. If the index is built in place using an exclusive algorithm and then is added to a table after the last LSN checkpoint, you may get a bad incremental backup using page tracking. Find more details in `PS-8032 <https://jira.percona.com/browse/PS-8032>`__ .
+
+2. The server internally generates a little data to maintain the page tracking data. You can purge this data before a full backup. Find more details in `Bug#105406 <https://bugs.mysql.com/bug.php?id=105406>`__ .
```

Comment:
Upstream has release the purge interface. so I think better to move this to other section. 
We also need to advice user how to purge. 

commit 0b524f8ba50d6b700db9727d83bdcbebeef79221
Author: Ingo Struewing <Ingo.Struewing@oracle.com>
Date:   Thu Nov 18 11:13:04 2021 +0100

    Bug#33529452 - MySQLbackup component doesn't have purge interface

Reply:
fixed